swerve100/src/test/java/team100/localization/VisionDataProviderTest.java

```diff
@@ -16,8 +16,10 @@
 import edu.wpi.first.math.geometry.Transform3d;
 import edu.wpi.first.math.geometry.Translation3d;
 import edu.wpi.first.wpilibj.DriverStation;
+import team100.control.DualXboxControl;
 
 public class VisionDataProviderTest {
+    private DualXboxControl m_control = new DualXboxControl();
```

Comment:
just make this a local variable in the test methods.  tests shouldn't have any shared variables.